Description: hardcode clean version number for deb packages of releases
 Somehow with our git-based version getter routine we get "-dirty" version
 numbers even when building clean packages from release tags. So it's easiest
 to simply hard code the version number.
Author: Tobias Megies
Bug: https://github.com/obspy/obspy/issues/2363
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/obspy/__init__.py
+++ b/obspy/__init__.py
@@ -38,7 +38,8 @@ import requests
 # don't change order
 from obspy.core.utcdatetime import UTCDateTime  # NOQA
 from obspy.core.util import _get_version_string
-__version__ = _get_version_string(abbrev=10)
+__version__ = _get_version_string(abbrev=10, dirty=False,
+                                  append_remote_tracking_branch=False)
 from obspy.core.trace import Trace  # NOQA
 from obspy.core.stream import Stream, read
 from obspy.core.event import read_events, Catalog
